scripts/items/beitetsu_box.lua

```diff
@@ -0,0 +1,16 @@
+-----------------------------------
+-- ID: 6184
+-- Beitetsu Box
+-- Breaks up a Beitetsu Box
+-----------------------------------
+local itemObject = {}
+
+itemObject.onItemCheck = function(target)
+    return xi.itemUtils.itemBoxOnItemCheck(target)
+end
+
+itemObject.onItemUse = function(target)
+    target:addItem(xi.item.BEITETSU, math.random(18, 33))
```

Comment:
Can you please cite the source for this range? BG has it as 15~30, Clopedia has 18~33.  I'd rather err towards the lower unless there is retail data to back up the increase.

Reply:
Only other thing I see is 15, 30 which I can reflect that